addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -133,6 +133,16 @@ $i: 1;
         }
     }
 
+    we-button.o_we_mobile {
+        width: 12px;
+        background-image: url('data:image/svg+xml;utf8,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="#{str-replace(#{$o-we-color-danger}, "#", "%23")}"><path d="M27.898 3.919l0.183 0.183c0.565 0.565 0.565 1.48 0 2.044l-21.935 21.935c-0.565 0.565-1.48 0.565-2.044 0l-0.183-0.183c-0.565-0.565-0.565-1.48 0-2.044l21.935-21.935c0.565-0.565 1.48-0.565 2.044 0z"/><path d="M22.322 22.547c0 0.374-0.303 0.677-0.677 0.677v0h-7.247l-6.529 6.53c0.476 0.431 1.111 0.695 1.807 0.695 0.001 0 0.002 0 0.004 0h12.643c1.496 0 2.709-1.213 2.709-2.709v0-15.15l-2.709 2.709zM16 28.643c-0.998 0-1.806-0.809-1.806-1.806s0.809-1.806 1.806-1.806c0.998 0 1.806 0.809 1.806 1.806v0c0 0.998-0.809 1.806-1.806 1.806v0z"/><path d="M9.678 4.937c0-0.374 0.303-0.677 0.677-0.677v0h11.289c0.136 0 0.263 0.041 0.368 0.111l-0.003-0.002 2.123-2.123c-0.477-0.431-1.112-0.695-1.808-0.695-0.001 0-0.002 0-0.003 0h-12.643c-1.496 0-2.709 1.213-2.709 2.709v0 15.15l2.709-2.709z"/></svg>');
+        background-repeat: no-repeat;
+        background-size: 1.2em;
+        &.active {
+            background-image: url('data:image/svg+xml;utf8,<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="#{str-replace(#{$o-we-sidebar-content-field-color}, "#", "%23")}"><path d="M22.322 1.55h-12.644c-1.496 0-2.709 1.213-2.709 2.709v0 23.481c0 1.496 1.213 2.709 2.709 2.709v0h12.644c1.496 0 2.709-1.213 2.709-2.709v0-23.481c0-1.496-1.213-2.709-2.709-2.709v0zM16 28.643c-0.998 0-1.806-0.809-1.806-1.806s0.809-1.806 1.806-1.806c0.998 0 1.806 0.809 1.806 1.806v0c0 0.998-0.809 1.806-1.806 1.806v0zM22.322 22.547c0 0.374-0.303 0.677-0.677 0.677v0h-11.289c-0.374 0-0.677-0.303-0.677-0.677v0-17.611c0-0.374 0.303-0.677 0.677-0.677v0h11.289c0.374 0 0.677 0.303 0.677 0.677v0z"/></svg>');
```

Comment:
The SVG should be an external ressource... not embed in the scss. The change of color can be done just like we do for all other custom SVG icon. Check the content width icons for example.

Also, the icons are still not aligned, neither horizontally, nor vertically, I should not have to tell you that:
![Screenshot from 2021-03-09 17-13-57](https://user-images.githubusercontent.com/10338094/110502286-5b30bf00-80fb-11eb-950b-019b6722252e.png)

Reply:
About the SVG as an external ressource:
- hard code the fill color (the one we don't want to do).
- Using it the same impl as `the content width icons`, I will need to add JS code to add the item (switch between two icons) as a child of the container (it accepts only one item by data-img).
  - I can append both icons(with and without a slash) and use the display mode to hide or show the icon that we want (JS code).
 My solution was to avoid JS code and use only the css.

Reply:
It's best to use the same system as for other icons, so yes share the JS code which loads those icons (like for content width). Indeed for the switch between two icons, you may need some extra code. Show/hide one of the two icons when the button is active or not, in css, is a solution. It may be done generically: the possibility to define two icons for a button, loaded in JS and then automatically shown or hidden depending on active or not (not only just for that one specific option).

Reply:
The problem with my icon is that it is rectangular (not square like other icons) 😅
<img width="338" alt="Capture d’écran 2021-03-10 à 11 25 35" src="https://user-images.githubusercontent.com/18035234/110615358-a26a8e80-8193-11eb-96e8-faa515829275.png">
<img width="338" alt="Capture d’écran 2021-03-10 à 11 25 30" src="https://user-images.githubusercontent.com/18035234/110615360-a3032500-8193-11eb-96c3-7f13c359ffa4.png">
<img width="338" alt="Capture d’écran 2021-03-10 à 11 25 18" src="https://user-images.githubusercontent.com/18035234/110615361-a3032500-8193-11eb-823e-422196cfd8a8.png">
<img width="338" alt="Capture d’écran 2021-03-10 à 11 25 12" src="https://user-images.githubusercontent.com/18035234/110615362-a39bbb80-8193-11eb-9a72-8dffb7223e28.png">